dces-drc-integration/src/main/java/uk/gov/justice/laa/crime/dces/integration/service/ContributionService.java

```diff
@@ -59,22 +62,19 @@ private void sendContributionsToDrc(List<ConcurContribEntry> contributionsList,
                 continue;
             }
 
-            // TODO: Send Contribution to DRC on line below:
-            boolean updateSuccessful = true; // hook in drc call here.
-            // handle response
-            // if successful/failure track accordingly.
+            boolean updateSuccessful = Boolean.TRUE.equals(drcClient.sendUpdate(createDrcDataRequest(contribEntry)));
```

Comment:
Is this used for null-safety? 
If so, is null return a valid scenario? Would a null return not be in the form of some form of exception, which should cascade upwards and be caught? 
The return is a Boolean, does this have to be boxed?